recipes/samhaplotag/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "SamHaplotag" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "700a3cdf1ffa7b4034e6e3f480398306772e5abf8e96ad286d5bc445d2bb976a" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/wtsi-hpag/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - clang>=9.0.1 # [linux] 
+    - meson>=0.57.1
+    - samtools >=1.10
```

Comment:
Should samtools not instead be present during run-time?

Reply:
> Should samtools not instead be present during run-time?

samtools isn't a strict runtime dependency and is only required to run a small test program after building (meson check), if it's a problem the check stage can be skipped and the dependency removed

Reply:
Ah, that's fine then, I just wanted to make sure it wasn't in the wrong section :)